src/main/resources/lib/menu.js

```diff
@@ -10,56 +10,56 @@ const globals = {
 
 /**
  * Returns the full breadcrumb menu path for the current content and site.
- * @param {Object} params - A JSON object containing the (optional) settings for the function.
+ * @param {Object} [params={}] - A JSON object containing the (optional) settings for the function.
  *   @param {Boolean} [params.linkActiveItem=false] - Wrap the active (current content) item with a link.
  *   @param {Boolean} [params.showHomepage=true] - Disable return of item for the site homepage.
  *   @param {String} [params.homepageTitle=null] - Customize (overwrite) the displayName of home/site link (if used). Common usage: "Home" or "Start".
  *   @param {String} [params.dividerHtml=null] - Any custom html you want appended to each item, except the last one. Common usage: '<span class="divider">/</span>'.
  *   @param {String} [params.urlType="Server"] - Control type of URL to be generated for menu items, default is 'server', only other option is 'absolute'.
- *   @param {String} [params.navigationAriaLabel=null] - The 'aria-label' attribute text on the '<nav>' element. This should be the name of the navigation, e.g "Breadcrumbs".
+ *   @param {String} [params.ariaLabel="breadcrumbs"] - The 'aria-label' attribute text on the '<nav>' element. This should be the name of the navigation, e.g "Breadcrumbs".
  * @returns {Object} - The set of breadcrumb menu items (as array) and needed settings.
  */
 exports.getBreadcrumbMenu = function (params = {}) {
-    let site = libs.portal.getSite();
-    let content = libs.portal.getContent() || site; // Fallback to site if there's no content (like in errorHandlers).
-
-    let breadcrumbItems = []; // Stores each menu item
-    let breadcrumbMenu = {}; // Stores the final JSON sent to Thymeleaf
-
     // Safely take care of all incoming settings and set defaults, for use in current scope only
     let settings = {
         linkActiveItem: params.linkActiveItem || false,
         showHomepage: params.showHomepage || true,
         homepageTitle: params.homepageTitle || null,
         dividerHtml: params.dividerHtml || null,
         urlType: params.urlType == "absolute" ? "absolute" : "server",
-        navigationAriaLabel: params.navigationAriaLabel || null,
+        ariaLabel: params.ariaLabel || "breadcrumbs",
```

Comment:
Er dette den eneste endringen som gjør det breaking (4.0.0-BETA)? Kunne i og for seg vurdert å beholde den på 3, ved å fortsatt fjerne dokumentasjonen til navigationAriaLabel men legge inn en fallback her og andre relevante steder: `ariaLabel: params.ariaLabel || params.navigationAriaLabel || "breadcrumbs",`

Reply:
🤷 Ja tror vi kan gjøre breadcrumb metoden none breaking med dette, men den blir med i 4.0 versjonen uansett? 
Er det ikke like greit å bare sette samme navn på breadcrumb og menu?

Reply:
Jo, gjerne det uansett. Det var bare et forslag om at hvis vi vil unngå breaking changes så lenge som mulig, og dette er det _eneste_ som bumper major til 4, så kunne bumpingen vært unngått sånn.

Reply:
eneste? 😨 Flere ting som er breaking som f.eks returnerer nå et object ikke en array.